src/components/Button/Button.tsx

```diff
@@ -275,7 +352,8 @@ const Button = forwardRef(
         {children}
         {rightIcon ? (
           <Icon
-            iconType={Icon.type.ICON_FONT}
+            // @ts-ignore
```

Comment:
why the ts-ignore? what's the hidden error?

Reply:
I was sure I add here a comment will recheck